jwst/tests_nightly/general/nirspec/test_extract2d_msa_8slits.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pytest
+import shutil
+from astropy.io import fits
+from jwst.extract_2d.extract_2d_step import Extract2dStep
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+def test_extract2d_nrs_msa(_bigdata):
+    """
+
+    Regression test of extract_2d step performed on NIRSpec fixed slit data.
+
+    """
+    input_files = [_bigdata+'/nirspec/test_extract_2d/msa/jw00038001001_01101_00001_NRS1_assign_wcs.fits',
+                   _bigdata+'/nirspec/test_extract_2d/msa/msa_configuration.fits'
+                   ]
+
+    try:
+        os.remove("extract2d2_output.fits")
+        for f in input_files:
+            os.remove(os.path.basename(f))
+    except:
+        pass
+
+    try:
+        # This is needed because we can't run the step with the
+        # MSA configuration file not in the current dir.
+        # When this is fixed, the copy can be removed.
+        for f in input_file:
+            shutil.copyfile(f, os.path.join(".", os.path.basename(f)))
+    except:
+        raise OSError("Could not copy inputs files")
+
+    Extract2dStep.call('jw00038001001_01101_00001_NRS1_assign_wcs.fits',
+                       output_file='extract2d2_output.fits'
+                       )
+    h = fits.open('extract2d2_output.fits')
+    href = fits.open(_bigdata+'/nirspec/test_extract_2d/msa/jw00038001001_01101_00001_NRS1_assign_wcs_extract_2d.fits')
+    '''
+    newh = fits.HDUList([h['primary'],h['sci',1],h['err',1],h['dq',1],
```

Comment:
This is a question for the person who wrote that test originally -- Why the commented code block? If it is not used going forward, it is less confusing to simply delete it.